earth_enterprise/src/fusion/portableglobe/gerewritedbroot.cpp

```diff
@@ -288,9 +302,17 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
           "                    Default: 8888\n"
           "   --kml_url_path:  Path in new url to prefix kml file name.\n"
           "                    Default: kml\n"
-          "   --use_ssl_for_kml:  Use https:// instead of http:// for\n"
+          "   --use_ssl_for_kml:  \n"
+          "                    Use https:// instead of http:// for\n"
           "                    accessing kml files.\n"
+          "                    Default: false\n"
+          "   --preserve_kml_filenames:\n"   
```

Comment:
Starting on line 267 it lists all of the options. We should add the new option there as well or just remove that section.

Reply:
Adding it is best, IMHO.  Thanks!